pkg/config/expand.go

```diff
@@ -710,11 +711,15 @@ func expandSimpleVariable(
 			if !slices.Contains(toMod.Outputs, ref.Name) {
 				toMod.Outputs = append(toMod.Outputs, ref.Name)
 			}
-			// TODO: expandedVariable = fmt.Sprintf("((var.%s_%s))", ref.Name, ref.ID)
-			return "", fmt.Errorf("%s: %s is an intergroup reference",
-				errorMessages["varInAnotherGroup"], context.varString)
+			switch fromMod.Kind {
+			case "terraform":
+				expandedVariable = fmt.Sprintf("((var.%s_%s))", ref.Name, ref.ID)
+			case "packer":
+				expandedVariable = "((TEST))"
+			}
+		} else {
```

Comment:
I would put the simple intragroup case first. It keeps the if -> else closer together and more readable.

Reply:
Will do.

Reply:
go-critic complained about the revised solution and advised using a `switch`. please see update with comments.

Reply:
looks better.